swarm/storage/localstore.go

```diff
@@ -208,7 +208,7 @@ func (ls *LocalStore) Migrate() error {
 						break
 					}
 				}
-				return valid
+				return !valid
```

Comment:
this should really be abstracted out as a separate function in localstore, why fiddle here

Reply:
agreed. extracted `isValid` as it is used at two places currently.